services/convert/attachment.go

```diff
@@ -16,12 +13,7 @@ func WebAssetDownloadURL(repo *repo_model.Repository, attach *repo_model.Attachm
 }
 
 func APIAssetDownloadURL(repo *repo_model.Repository, attach *repo_model.Attachment) string {
```

Comment:
Do we even need the `repo` param anymore now?

Reply:
I think these function could be removed. To keep this PR small, I need another refactor PR to do that.